changelog.d/scriv.ini

```diff
@@ -0,0 +1,5 @@
+[scriv]
```

Comment:
I didn't do it because putting the conf file in `changelog.d` avoids the need to create a dummy file to keep the directory when there are no changelog fragments, but `pyproject.toml` usage is so widespread nowadays that I just changed to it (and added a `changelog.d/README.md` file).